language_model/rnn_train.py

```diff
@@ -0,0 +1,104 @@
+import sys
+import sqlite3
+import paddle.v2 as paddle
+import ptb
+import gzip
+
+
+def rnn_net(dict_dim, is_generating=False):
```

Comment:
`is_generating` is not used and it will not be a parameter for this function. I think it should be removed.

In the `language model` example, a generating is required and is expected to show how to generate a sequence by using a trained language model, which is quite a different from training, but generating is not implemented yet in this pr.

1. the encoding RNN: load the trained model to encode the given input sequence and output the sentence encoding (a dense vector) of the input sentence.
2. the generate RNN: initialized by the sentence encoding calculated by the first part RNN, every time, it takes the most probable words in last time step as inputs.

Using a trained RNN language model to generate a sequence is not straightforward. It begins by given an input sentence, and then generate the following sentence. You should split the original trained RNN model into two parts, which have the same architecture and share the trained parameter, but have different effects: the first part is to encode the input sequence, while the second part is to generate.

To generate a sequence, you have to implement the OneWay search or BeamSearch (optional) algorithm. I recommend using `recurrent_layer_group` to implement the RNN language model to avoid implementing OneWay search or BeamSearch yourself. Currently, in PaddlePaddle, generating algorithms like OneWay search or BeamSearch is only implemented in `recurrent_layer_group`.